history/cultures/azerothian.txt

```diff
@@ -23,6 +23,8 @@
 	discover_innovation = innovation_casus_belli 
 	discover_innovation = innovation_currency_01 
 	#
+	discover_innovation = innovation_elite_ranks
```

Comment:
It's not discovered though. Many other human cultures have the same problem.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/104135087-3217da80-53a7-11eb-8f80-7baab3233455.png)

</details>